jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -75,6 +75,7 @@
         frequency: 'H 0 * * *'
         scan: DISABLED
         timeout: 110
+        soak-repos: '--repo=k8s.io/kubernetes --repo=k8s.io/release'
```

Comment:
Please define `soak-repos` in every job in this file (although see prior suggestion about making this two vars, one per repo)

Also, please update `jenkins/bootstrap_test.py` to ensure that `soak-repos` is defined in every job in this file.